src/DynamoDbModel.php

```diff
@@ -247,12 +247,29 @@ public function getClient()
     /**
      * @return array
      */
-    public function getCompositeKey()
+    public function getCompositeKeyName()
     {
         return $this->compositeKey;
     }
 
     /**
+     * @return array
+     */
+    public function getCompositeKey()
```

Comment:
Not super keen on this because there is no longer a symmetry  of getter and setter for property `$compositeKey`. I would prefer to have this logic in the method `getKey()`.

Reply:
getCompositeKey() and getCompositeKeyName() are named using the conventions in Eloquent for getKey() and getKeyName()

Reply:
Yes, but now it breaks the pattern of getter and setter. We can override `getKey` and `getKeyName` to handle composite keys.

Reply:
OK, I can see that.  Unfortunately, I don't have the time this week to work on these changes.